exporters/otlp/otlp_test.go

```diff
@@ -104,17 +118,23 @@ func newExporterEndToEndTest(t *testing.T, additionalOpts []otlp.ExporterOption)
 	spans := mc.getSpans()
 
 	// Now verify that we received all spans.
-	if got, want := len(spans), m; got != want {
+	if got, want := len(spans), m*2; got != want {
 		t.Fatalf("span counts: got %d, want %d", got, want)
 	}
 	for i := 0; i < 4; i++ {
```

Comment:
Should this be 0 -> m*2?
```suggestion
	for i := 0; i < m*2; i++ {
```

Reply:
yes. changed.